tasks/main.yml

```
@@ -105,30 +80,32 @@
     state: present
     login_user: "{{ database_root_user }}"
     login_password: "{{ database_root_password }}"
```

Password may be unset. This would handle this case.

```suggestion
    login_password: "{{ database_root_password | default(omit) }}"
```